packages/graphql-language-service-interface/src/__tests__/getAutocompleteSuggestions-test.ts

```diff
@@ -0,0 +1,321 @@
+/**
```

Comment:
noting the path for these files wasn't rewritten, and there really weren't that many changes. not a huge deal also not ideal